readme.md

```diff
@@ -113,6 +113,33 @@ function listItem(n: number) {
 </ul>
 ```
 
+#### Using a helper template like an element
+
+Want a helper component? Create a function that implements CustomElementHandler and you can call it like an HTML element. 
+
+```typescript
+interface CustomElementHandler {
```

Comment:
this can be imported right? `import { CustomElementHandler } from 'typed-html'`

Reply:
Yep. Checked on a project I had with typed-html as an NPM dependency

Reply:
So should we remove the interface definition here? I wouldn't want people defining an interface they don't control. Can we replace this interface definition here with the import statement?

Reply:
Yeah, might be a good idea. I originally added it to show why the function parameters were defined as they were. But I think the type annotation on the lambda gets the same point across.

I'll also add Attributes like so  `import { Attributes, CustomElementHandler } from "typed-html"`

Reply:
Sounds good 👍

Reply:
Hi. My bad, I did a rebase again out of habit. You might not have noticed that the latest readme was updated to remove the interface declaration.